src/card/block.spec.js

```diff
@@ -0,0 +1,180 @@
+// card test
+jest.setTimeout(30000)
+import {
+    createNewPost,
+    enablePageDialogAccept,
+    getEditedPostContent,
+    insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+import {
+    clickElementByText,
+    blockStartup,
+    clickRadio,
+    clickCheckbox
+} from '../test-helpers'
+
+const block = {blockTitle: 'Custom Card', blockName: 'purdue-blocks/card'}
+
+describe( '🔬 Card Block', () => {
+    beforeAll( async () => {
+        await enablePageDialogAccept();
+    } );
+    beforeEach( async () => {
+        await createNewPost();
+    } );
+
+    test( '🔬 Block should be available.', async () => {
+        await insertBlock( 'Custom Card' )
+
+        // tests that the block is properly inserted and matches the existing snapshot
+        expect(await page.$('[data-type="purdue-blocks/card"]')).not.toBeNull()
+        expect( await getEditedPostContent()).toMatchSnapshot()
+    } )
+
+    describe( '🔬 Block Editor Fields', () => {
+        test( '🔬 Button should open Media Library.', async () => {
```

Comment:
Just put the microscope on the describes, not all the tests.

Reply:
I found it was still a little hard to read in the snapshot so I added them.

Reply:
Yeah you're not wrong, maybe we should find a different one for the tests just to differentiate

Reply:
what about a magnifying glass 🔎

Reply:
or a page 📄

Reply:
or maybe a test tube 🧪

Reply:
I think all of them work. Maybe we just pick 🔎

Reply:
Haha okay go ahead and use that one and I can go back through the tests I've already made and add it

Thanks

Reply:
Done. Thanks!